velox/functions/sparksql/tests/DecimalVectorFunctionTest.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/functions/sparksql/tests/SparkFunctionBaseTest.h"
+
+using namespace facebook::velox::test;
+
+namespace facebook::velox::functions::sparksql::test {
+namespace {
+class DecimalVectorFunctionlTest : public SparkFunctionBaseTest {};
```

Comment:
rename file and class to UnscaledValueFunctionTest

Reply:
Um, later on, we will add some decimal-related functions, such as `make_decimal` and `check_overflow`. These functions are part of a series, similar to the `DateTimeFunctions.h` file we currently have, which contains date-related functions. Therefore, we would like to place these functions in a single file called `DecimalVectorFunctions.h`. Would it be too much to generate a separate file for each function?